### DISCUSSION TITLE
how to move the drawSvg.elements integrally

### DISCUSSION THREAD

**Discussion by lythhh** [cduck/drawSvg#72] [Q&A] [answered]
I make a simple drawSvg.elements(a triangle), what i want to do is to move the triangle a certain distance, and finally form a series of ordered triangles. So i want to konw is there any function can help me? Thank you very much 
 the code is as follows:
import drawSvg as draw
def mm2pt(x):
    return 2.835*x
dw=mm2pt(3)
dh=mm2pt(3)
d = draw.Drawing(dw, dh, origin=(0, 0), displayInline=False)
r = draw.Lines(dw/4,dh/4,dw/4*3,dh/4,dw/2,dh/4*3,close=True,fill='none',stroke='black',stroke_width=mm2pt(0.1))
r.appendTitle("角砾")
d.append(r)
d.setPixelScale(5) 
d
#def move(drawSvg,w,h): #i want to construct a function with a drawSvg object and certain distance on x and y, to realize the movement of drawSvg objects.
![1](https://user-images.githubusercontent.com/48010500/162580899-7600972f-0df2-4d9d-bfd5-d4a6190d5c89.PNG)

**Comment by cduck** [comment, marked as answer]
I'm glad you are enjoying the library and I am happy to clarify the usage of the library but please do a bit more of your own learning about how SVGs work before asking here.  I am only one person, doing this in my spare time.  [stackoverflow.com](https://stackoverflow.com/search?q=drawSvg) will probably be more helpful.
